src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CountDocumentsCommandResolver.java

```diff
@@ -17,11 +20,21 @@ public class CountDocumentsCommandResolver extends FilterableResolver<CountDocum
     implements CommandResolver<CountDocumentsCommand> {
 
   private final OperationsConfig operationsConfig;
+  private final MeterRegistry meterRegistry;
+  private final DataApiRequestInfo dataApiRequestInfo;
+  private final JsonApiMetricsConfig jsonApiMetricsConfig;
 
   @Inject
-  public CountDocumentsCommandResolver(OperationsConfig operationsConfig) {
+  public CountDocumentsCommandResolver(
+      OperationsConfig operationsConfig,
```

Comment:
Could any of these instead be passed and accessed via `CommandContext`?

Reply:
`MeterRegistry` is an application scoped bean. Not sure if it belongs in `CommandContext`. Any reason other than injected in many class to move this to CommandContext?

Reply:
No, just convenience of not having to add field to store during construction, since it's needed for resolve().
So not a big deal; it'd just be convenient to access configuration via context, instead of injecting a bunch.

So no point pursuing for now.